sahi/annotation.py

```diff
@@ -14,6 +13,11 @@
 )
 from sahi.utils.shapely import ShapelyAnnotation
 
+try:
+    from pycocotools import mask as mask_utils
+    use_rle = True
+except:
```

Comment:
exception should be defined:

```suggestion
except ImportError:
```